Header header

uint8 MODE_CONTROL_FORCE_ENU = 0
uint8 MODE_CONTROL_FORCE_BODY = 1
uint8 MODE_CONTROL_FORCE_BODY_UP = 2
uint8 MODE_CONTROL_ALTHOLD_FORCE_ENU = 3
uint8 MODE_CONTROL_ALTHOLD_FORCE_BODY = 4
uint8 MODE_CONTROL_ALTHOLD_FORCE_BODY_UP = 5
uint8 position_control_mode

uint8 MODE_CONTROL_ATT = 6
uint8 MODE_CONTROL_RATES = 7
uint8 MODE_CONTROL_RP_ATT_Y_RATE = 8
uint8 MODE_CONTROL_YAWRATE = 9
uint8 MODE_CONTROL_ZERO_ATT = 10
uint8 attitude_control_mode